src/api/account_api.js

```diff
@@ -443,6 +444,7 @@ module.exports = {
                     },
                     identity: { $ref: 'common_api#/definitions/access_key' },
                     secret: { $ref: 'common_api#/definitions/secret_key' },
+                    azure_log_access_keys: { $ref: 'common_api#/definitions/azure_log_access_keys' },
```

Comment:
you added the support in the API, but I didn't see the use of it in the implementation in account_server. same foe `check_external_connection`